.travis.yml

```diff
@@ -53,9 +53,6 @@ env:
         - CONF=""
         - CONF="--disable-abi-compat"
         - CONF="--enable-deprecated"
-        - CONF="--enable-schedule-sp"
-        - CONF="--enable-schedule-iquery"
-        - CONF="--enable-schedule-scalable"
         - CONF="--enable-dpdk-zero-copy"
```

Comment:
Run time selection is good addition and can be even the default behavior. Anyway, I'd like to keep the build time selection also, since usually one application uses the same scheduler always. Fixing the scheduler at build time avoids accidentally usage of different schedulers in different servers (or VMs or test environments) and thus avoid problems/debugging caused by that.

Reply:
Perhaps `--enable-schedule-dynamic` (default) to indicate run time selection that can be overridden by a specific static selection at `./configure` time via the other `--enable-schedule-xxx` options?

Reply:
This would mean that we have now 4 options to test, instead of just one. What about renaming 'default' scheduler to some sensible name and adding `--with-scheduler-default=FOO`? Then we will have just one combination to test.

Reply:
There can be just one config option: --with-scheduler=foo. When that's defined then "foo" would be used (fixed at build time). If --with-scheduler is not found, the value of ODP_SCHEDULER=foo environment variable is used (run time selection is the default behavior). If the variable is not found then the default scheduler is used.

Reply:
@psavol this would also require seprate compile testing of `--with-scheduler` option. I'm thinking about `--with-scheduler-default` option with the following behaviour:
 - If environment option is given, it selects scheduler/queue implementation
 - If no environment option is given, compile-time option selects the scheduler/queue
 - If no compile-time option was given, we use some sensible default (basic for now, maybe scalable in future).

Reply:
@psavol Is @lumag's latest suggestion OK with you?